### DISCUSSION TITLE
라인차트의 2번째 Y축

### DISCUSSION THREAD

**Discussion by JunLee26** [nhn/tui.chart#697] [Q&A]
안녕하세요
#218 이슈의 연장 질문입니다.

라인차트에서 배열형태로 Y축을 줘서 min, max로 scale처리를 한다고 해도 첫번째 Y축의 종속되는것 같네요.
라인(data.series)이 3,4개 이상 될 수도 있는데.. 원하는 Y축(좌/우)을 선언할 수는 없나요?
lineColumn, lineArea 차트처럼 data에서 원하는 축을 선택할 수 없을까요?
yAxis에 chartType대신 chartID를 선언해서 사용할 수 있으면 좋을텐데요..

chart.js에서는 yAxisID처럼 Y축의 id값을 선택하게 하는데요.
이런 기능은 현재 없는건가요?

**Comment by jwlee1108** [comment]
@JunLee26 

안녕하세요. 답변이 많이 늦어져 죄송합니다.
현재 차트 타입과 관련되어 차이를 둘 수 있고 같은 차트 내 시리즈 별로 축을 다르게 종속시키는 방법은 없습니다. 
[링크](https://github.com/nhn/tui.chart/blob/main/docs/en/common-axes.md#secondary-y-axis)를 참고 부탁드려요.

정확하게 일정을 말씀드릴 순 없지만 해당 기능은 저희가 갖고 있는 과제 중 하나입니다.

---

### translated
Hello. I'm sorry for the delay in answering.
Currently, there can be differences in chart types, and there is no way to depend on axes differently for each series in the same chart.
Please refer to the [link](https://github.com/nhn/tui.chart/blob/main/docs/en/common-axes.md#secondary-y-axis).

I can't tell you the exact schedule, but that function is one of our challenges.
